src/Components/Card.js

```diff
@@ -1,29 +1,27 @@
 import React from 'react';
+import MenuNames from '../Menu.json';
+import './Card.css';
+
 
 class Card extends React.Component {
 
    constructor(props) {
       super(props);
       this.cardStyle = {
-         fontFamily: "cursive",
+         // fontFamily: "cursive",
```

Comment:
Is this being kept for a future usage? If not, you may want to remove it. Seems like you're handling the font-styles efficiently in the css files.

Reply:
Thanks for all the feedback Jacob! I'll make the changes and have another pull request (or two) today.